pkgs/top-level/stage.nix

```diff
@@ -117,11 +117,17 @@ let
     lib.optionalAttrs allowCustomOverrides
       ((config.packageOverrides or (super: {})) super);
 
-  # Convenient way to reference cross
-  # Used in aliases for now but should not be used in Nixpkgs.
-  cross = self: super: { pkgsCross = lib.mapAttrs (n: crossSystem:
-                                     nixpkgsFun { inherit crossSystem; })
-                                     lib.systems.examples; };
+  # Convenience attributes for instantitating nixpkgs. Each of these
+  # will instantiate a new version of allPackages. They map example
+  # attributes to their own thing.
+  extraPkgs = self: super: {
+     pkgsCross = lib.mapAttrs (n: crossSystem:
+                              nixpkgsFun { inherit crossSystem; })
+                              lib.systems.examples;
+     pkgsLocal = lib.mapAttrs (n: localSystem:
```

Comment:
Please call these something more descriptive (`pkgsForOtherSystems` or something). "local" sounds something totally different like these packages won't ever use remote building.

Reply:
It is just based on crossSystem vs localSystem. I would like to avoid the confusion but not sure how useful it is with a long name like that. Maybe pkgsLocalSystem & pkgsCrossSystem ?

Reply:
Maybe `pkgsForOtherSystems.example.{cross,native}`?

Reply:
Hmmm... maybe. What about pkgsNative instead of pkgsLocal? I guess that would still invite confusion with something like buildPackages though.

Reply:
Maybe `nativePkgsForOtherSystems.<whatever>` and `crossPkgsForOtherSystems.<whatever>`?